libr/asm/p/asm_mips_cs.c

```diff
@@ -25,9 +25,11 @@ static int disassemble(RAsm *a, RAsmOp *op, const ut8 *buf, int len) {
 			mode |= CS_MODE_MIPS3;
 //		} else if (!strcmp (a->cpu, "gp64")) {
 //			a->bits = 64;
+		} else if (!strcmp (a->cpu, "64v2")) {
+			mode |= CS_MODE_MIPS32;
```

Comment:
uhm, this requires asm.bits to be 64 to make sense. what about making mode |= CS_MODE_MIPS32 | CS_MODE_64 here?